src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -72,6 +85,19 @@ class OutcomeCardContainer extends Component {
     });
   };
 
+  isEditing = () => {
+    return !this.state.activeEditingClients.isEmpty()
+  }
+
+  trackEditingClient = (uid, isEditing) => {
+    this.setState((curState) => {
```

Comment:
Just a note - I spent a while debugging some races where the current state (`this.state`) used to compute the next state was "old" due to how `setState` is async. Using the functional `setState` API allows updates that rely on current state to see the state which the state update queue sees.